src/main.ts

```diff
@@ -48,11 +48,11 @@ export default async function run() {
 
 function getInputs(): IActionInputs {
     core.debug('Get action inputs.');
-    let serverName = core.getInput('server-name', { required: true });
     
     let connectionString = core.getInput('connection-string', { required: true });
     let connectionStringBuilder = new SqlConnectionStringBuilder(connectionString);
```

Comment:
Won't it be better to have these a couple of lines below (_post the getInput calls_)? 
The intent here is to make it more readable